packages/core/src/browser/style/notification.css

```diff
@@ -22,7 +26,7 @@
     display: flex;
     font-size: calc(var(--theia-ui-font-size0) * 0.8);
     font-weight: 500;
-    height: calc(var(--theia-private-horizontal-tab-height) * 0.7);
```

Comment:
We need to check all places where `--theia-private-horizontal-tab-height` was used and make sure that they look nice.

Reply:
I've checked that last restyling PR did not change other variables.

Reply:
looking at the code for the variable the following are updated:
1. search box (for tree widgets)
2. `.theia-source-breakpoint .line` and `.theia-debug-stack-frame .line`

And previously the `notification-count`.

Reply:
the search box for the tree widgets does look fine, I'll have to check the debug styling.

Reply:
It is used in mini browser as well: https://github.com/theia-ide/theia/blob/8f9ba43508320675621bd98e4397a316f16aa292/packages/mini-browser/src/browser/style/index.css#L42-L51

It looks very thick compare to how it used to look.